tck/src/main/java/org/eclipse/microprofile/openapi/tck/AirlinesAppTest.java

```diff
@@ -549,6 +549,8 @@ public void testSecurityRequirement(String type) {
                 allOf(aMapWithSize(2),
                         hasEntry(equalTo("userApiKey"), empty()),
                         hasEntry(equalTo("userBearerHttp"), empty()))));
+
+        vr.body("paths.'/zepplins'/{id}/delete.security.mutualTLSScheme.scopes", equalTo("zepplinScope"));
```

Comment:
I think this should be:
```suggestion
        vr.body("paths.'/zepplins/{id}'.delete.security.mutualTLSScheme.scopes", contains("zepplinScope"));
```

It's possible that the quoting and `/` is equivalent, I don't fully understand the query syntax.

However, based on [the spec](https://spec.openapis.org/oas/v3.1.0.html#patterned-fields-2), the value should be a list.